integration/otlp_ingestion_test.go

```diff
@@ -72,4 +74,18 @@ func TestOTLPIngestion(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, model.ValMatrix, rangeResult.Type())
 	require.Equal(t, expectedMatrix, rangeResult.(model.Matrix))
+
+	// Push series with histograms to Mimir
+	series = []prompb.TimeSeries{
```

Comment:
Does OTLP also use prompb? I don't know at what stage the translation happens, should Mimir do the translation of histograms it by taking in OTel exponential histograms?

Reply:
The translation happens after Mimir OTLP endpoint receives OTLP payload that might contain exp. histograms. The translation is done with https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/pkg/translator/prometheusremotewrite that coverts `pmetric.Metrics` to the `prompb.TimeSeries`.